### PR TITLE
overlays margins

### PR DESCRIPTION
**Pull request by AlicVB**
- ensure that we have homogeneous margins in thumbnails overlays, esp. between bottom and stars, stars and text, text and image.
- express image margins in per thousand instead of percentage. That way we can increase a little accuracy as dot numbers are not handle in margins.

@Nilvus : can you have a look : this change slightly margins and so positions. esp. Thanks

### CODE REVIEW

**Comment by Nilvus** [comment]
Will see that tomorrow now. @TurboGit: please wait my review before merging that one. Thanks.

File: data/themes/darktable.css [hunk 1]
```diff
@@ -1907,12 +1907,12 @@ spinbutton>button
 #thumb_image
 {
   border: none;
-  margin: 2px; /* not real pixels, but percentage of the thumb size */
+  margin: 20px; /* not real pixels, but per thousand of the thumb size */
```
Comment (Nilvus) [inline data/themes/darktable.css:1910]: Increase to 25px for better margins and aerate thumbnail

File: data/themes/darktable.css [hunk 2]
```diff
@@ -1907,12 +1907,12 @@ spinbutton>button
...
 }
 
 .dt_thumbnails_2 #thumb_image /* update consistently margins if big thumbnails (less images per row) are shown */
 {
-  margin: 1.5px;
+  margin: 15px;
```
Comment (Nilvus) [inline data/themes/darktable.css:1915]: Increase to 20px for better margins and aerate thumbnail

**Comment by TurboGit** [comment]
Thanks guys for all the work on this. Merging now.

**Pull request merged**